src/noc/containers/noc-check-your-answers/noc-check-your-answers.component.html

```diff
@@ -16,7 +16,8 @@
   <ng-container *ngFor="let qAndA of qAndA$ | async">
     <div class="govuk-summary-list__row">
         <div class="govuk-summary-list__key">{{ qAndA.question_text | async }}</div>
-        <div class="govuk-summary-list__value">{{ qAndA.value }}</div>
+        <div class="govuk-summary-list__value">{{ (qAndA.question_type| async) === 'DateTime'?
```

Comment:
This would be cleaner with a pipe, rather than inline transformation within the binding.

Reply:
move that to a pipe now